src/pages/planters/[planterid].js

```diff
@@ -59,6 +59,13 @@ const useStyles = makeStyles()((theme) => ({
       marginBottom: theme.spacing(14),
     },
   },
+  profileImg: {
+    maxHeight: '764px',
+    borderRadius: '16px',
+    [theme.breakpoints.down('md')]: {
+      height: '332px',
+    },
+  },
```

Comment:
@jeremydthomas we are abanding the class style to do the CSS, the recommended way is `sx`

So this case, you can put it on the img like this: (I'm not sure this is fully correct)

```
<Box 
  sx={{
      "& img": {
          maxHeight: [332, 764],
...
```

Reply:
Another problem, looks the height is too big?

<img width="348" alt="image" src="https://user-images.githubusercontent.com/5744708/184062950-33307e61-cca1-4688-9a7b-bf6066463094.png">


The figma is here: https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=4819%3A14477

Reply:
Also, there is a mobile version for the same page, please check there to get the size

Reply:
@dadiorchen is the tree image that is there now, temporary? Because if I use those figma dimensions like that, it looks absolutely horrible

Reply:
> @jeremydthomas we are abanding the class style to do the CSS, the recommended way is `sx`
> 
> So this case, you can put it on the img like this: (I'm not sure this is fully correct)
> 
> ```
> <Box 
>   sx={{
>       "& img": {
>           maxHeight: [332, 764],
> ...
> ```

sounds good

Reply:
If i change the src to this

```
          <img src={`${router.basePath}${planterBackground}`} alt="profile" />
```

instead of this 
```
          <img src={backgroundPic} alt="profile" />
```

I get whats in the figma


![image](https://user-images.githubusercontent.com/63400531/184065235-1e666268-7eca-475f-aa78-167f2d694bc6.png)

Reply:
@jeremydthomas take a look at this: https://stackoverflow.com/questions/3029422/how-do-i-auto-resize-an-image-to-fit-a-div-container